storage/local/frankenstein.go

```diff
@@ -0,0 +1,181 @@
+package local
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/prometheus/common/log"
+	"github.com/prometheus/common/model"
+
+	"github.com/prometheus/prometheus/frankenstein/wire"
+	"github.com/prometheus/prometheus/storage/metric"
+)
+
+const (
+	flushCheckPeriod = 1 * time.Minute
+)
+
+// Ingestor deals with "in flight" chunks.
+// Its like memseriesstorage, but simplier.
```

Comment:
just a couple of typos :) "Its like memseriesstorage, but simplier." -> "It's like MemorySeriesStorage, but simpler."